runtime/fp8_support.cu

```diff
@@ -195,8 +207,19 @@ __device__ __inline__ __bfloat __e4m32bfloat(const __e4m3 h) {
   return val;
 }
 
+__device__ __inline__ __e4m3 operator|(const __e4m3 x, const __e4m3 y) {
```

Comment:
Not exactly sure if this is reasonable. I don't know why `memcpy` is extensively used for these fp8 types. 

Pinging @jjsjann123

Reply:
This does look cleaner :)

Reply:
I thought there may be some magic with `memcpy`.